### PR TITLE
Update travis config

### PR DESCRIPTION
**Pull request by mimmi20**

### CODE REVIEW

File: .php_cs [hunk 1]
```diff
@@ -12,7 +12,7 @@ return Symfony\CS\Config\Config::create()
     ->level(\Symfony\CS\FixerInterface::PSR2_LEVEL)
     ->fixers(
         array(
-            'unalign_double_arrow ',
+            'align_double_arrow',
```
Comment (asgrim) [inline .php_cs]: This conflicts (space was removed), and it should stay as `unalign` imo; aligning creates extra whitespace diff noise :s

File: .travis.yml [hunk 2]
```diff
@@ -6,47 +6,56 @@ php:
...
-  matrix:
-    - PREFER_LOWEST="--prefer-lowest"
-    - PREFER_LOWEST=""
+  - COMPOSER_FLAGS="--prefer-lowest"
+  - COMPOSER_FLAGS=""
 
 matrix:
+  include:
+    # Use the newer stack for HHVM as HHVM does not support Precise anymore since a long time and so Precise has an outdated version
+    - php: hhvm
```
Comment (asgrim) [inline .travis.yml]: TBH I'm quite happy to drop HHVM support

File: tests/bootstrap.php [hunk 3]
```diff
@@ -1,29 +0,0 @@
-<?php
```
Comment (asgrim) [inline tests/bootstrap.php]: This not needed any more?

Comment (mimmi20) [inline tests/bootstrap.php]: No. We are loading `vendor/bootstrap.php` as the bootstrap file in the phpunit configuration. The memory limit is there too.
Why we sat here an monolog Logger, I don't remeber.

**Comment by mimmi20** [comment]
replaced by PR #184.

**Pull request closed without merging**